app/Http/Controllers/Service/Admin/CentresController.php

```diff
@@ -99,4 +100,47 @@ public function store(AdminNewCentreRequest $request)
             ->route('admin.centres.index')
             ->with('message', 'Centre ' . $centre->name . ' created');
     }
+
+    /**
+     * Show the form for editing a Centre.
+     *
+     * @return Factory|View
+     */
+    public function edit($id)
+    {
+        $centre = Centre::where('id', $id)->first();
```

Comment:
laravel has a shortcut for this as Centre::find($id)

[ As well as findOrFail(), which I see you use below to force an exception if it cant find one there's findOrCreate($id, [array of attributes]) which will _create_ one if it can't find it ]

Reply:
Changed.